Tests/UnitTests/Misc/SandboxEnvironmentDetectorTests.swift

```diff
@@ -54,41 +54,85 @@ class SandboxEnvironmentDetectorTests: TestCase {
         ) == false
     }
 
-    func testIsSandboxIfReceiptIsProductionAndNotMAS() throws {
+    func testIsSandboxIfReceiptIsNotProduction() throws {
         expect(
             SystemInfo.with(
                 macAppStore: false,
-                receiptEnvironment: .production
+                receiptEnvironment: .sandbox
             ).isSandbox
         ) == true
     }
 
-    func testIsSandboxIfReceiptIsNotProductionAndNotMAS() throws {
-        expect(
-            SystemInfo.with(
-                macAppStore: false,
-                receiptEnvironment: .sandbox
-            ).isSandbox
-        ) == true
```

Comment:
this test and the next one are technically still there, but now in a more complete test that also verifies that we indeed use MAS when needed and skip it when not needed